AI-and-Analytics/Getting-Started-Samples/IntelJAX_GettingStarted/README.md

```diff
@@ -74,16 +62,14 @@ source <custom_path>/bin/activate
 
 For the system with Intel CPU:
 ```
-conda activate tensorflow
-```
-For the system with Intel GPU:
-```
-conda activate tensorflow-gpu  
-```
+conda activate jax
+``` 
+
 **4. Clone the GitHub repository**
 ``` 
-git clone https://github.com/oneapi-src/oneAPI-samples.git
-cd oneAPI-samples/AI-and-Analytics/Getting-Started-Samples/IntelTensorFlow_GettingStarted
+git clone https://github.com/google/jax.git
```

Comment:
```suggestion
wget https://raw.githubusercontent.com/jax-ml/jax/refs/heads/main/examples/spmd_mnist_classifier_fromscratch.py
```

Reply:
might be easier?

Reply:
We need the entire repo  since the sample code has  got some dependencies.